modules/custom/express_status/js/express-status.js

```diff
@@ -0,0 +1,58 @@
+(function ($) {
+  Drupal.behaviors.expressStatus = {
+    attach: function (context, settings) {
+
+      var statusID = Drupal.settings.express_status.statusio_id;
+      var statusAPI = Drupal.settings.express_status.statusio_apikey;
+
+      var sp = new StatusPage.page({page: statusID, apiKey: statusAPI});
+      sp.summary({
+        success: function (data) {
+          statusioDisplay(data);
+        }
+      });
+      function formattedDate(d) {
+        let month = String(d.getMonth() + 1);
+        let day = String(d.getDate());
+        const year = String(d.getFullYear());
+        var time = d.toLocaleString('en-US', {
+          hour: 'numeric',
+          minute: 'numeric',
+          hour12: true
+        });
+        return `${month}/${day}/${year} ${time}`;
+      }
+
+      function statusioDisplay(data) {
```

Comment:
Possibly use data.components to list out all services that are "only_show_if_degraded = false" and set services like SAML and Salesforce based on modules being enabled/disabled on sites.